src/mlpack/core/util/CMakeLists.txt

```diff
@@ -11,15 +12,18 @@ set(SOURCES
   cli_deleter.cpp
   cli_impl.hpp
   deprecated.hpp
+  gitversion.hpp
   hyphenate_string.hpp
   is_std_vector.hpp
   log.hpp
   log.cpp
   mlpack_main.hpp
   nulloutstream.hpp
+  param.hpp
   param_checks.hpp
   param_checks_impl.hpp
   param_data.hpp
+  param_data_impl.hpp
```

Comment:
This file is actually unused (it's from a previous setup of the parameter handling code) and we can just remove it.